src/library-authoring/author-library/index.scss

```diff
@@ -1,3 +1,21 @@
+.library-authoring-wrapper {
+  margin: 0;
+  font-size: 1.4rem;
+
+  .paging-header {
+    display: flex;
+    align-items: center;
+    margin-bottom: 0.5rem;
+    p {
```

Comment:
Please, add empty line before this code block

Reply:
added